pkg/pillar/dpcmanager/verify.go

```diff
@@ -148,9 +148,8 @@ func (m *DpcManager) runVerify(ctx context.Context, reason string) {
 		}
 	}
 
-	// If there are port level errors in current selected DPC, we should mark
-	// it for re-test during the next TestBetterTimer invocation.
-	if m.dpcList.CurrentIndex != 0 || m.deviceNetStatus.HasErrors() {
+	// Try to get back to the latest and the highest priority DPC.
```

Comment:
Does the dpcTestTimer ensure that we clear out old errors on unused ports?

Reply:
yes